CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 _The versioning refers to the React component build._
 
+#### v2.0.4 (2017-09-06)
```

Comment:
I would date this as the date of the version merge on npm, as that's when the version starts being available — so, today?